providers/apns/src/lib/apns.provider.ts

```diff
@@ -25,7 +27,7 @@ export class APNSPushProvider implements IPushProvider {
         keyId: config.keyId,
         teamId: config.teamId,
       },
-      production: process.env.NODE_ENV === 'prod',
+      production: config.production,
```

Comment:
What could be the outcome if a user enables the production switch while testing in the dev environment?

Reply:
it uses production instead of sandbox for apns at Apple (I guess it is apple that is hosting this)